client/src/Components/Alerts/ListAlert.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const ListAlert = props => {
+
+    if(props.alerts.length === 0){
+        return (
+            <div class="card card-primary card-outline">
+                <div class="card-body">
+                <h5 class="card-title">No alerts found.</h5>
+                </div>
+            </div>
+        );
+    }
+
+    return (
+    <div>
+       {props.alerts.map(alert =>(
+        <div className="card card-primary card-outline" key={alert._id}>
+            <div className="card-body">
+                <h5 className="card-title">Symbol: {alert.symbol}</h5>
+                <p className="card-text">
+                  Send me a email if price is {alert.condition === 1 ? 'more than' : 'less than'} {alert.value}.
```

Comment:
Can we use some constants here and throughout the app like `=== ALERT_CONDITIONS.MORE_THAN`?